ksmt-symfpu/src/main/kotlin/org/ksmt/symfpu/operations/utils.kt

```diff
@@ -0,0 +1,229 @@
+package org.ksmt.symfpu.operations
+
+import org.ksmt.KContext
+import org.ksmt.expr.KBitVecValue
+import org.ksmt.expr.KExpr
+import org.ksmt.sort.KBoolSort
+import org.ksmt.sort.KBvSort
+import org.ksmt.sort.KFpSort
+import org.ksmt.utils.BvUtils.bvOne
+import org.ksmt.utils.BvUtils.bvValue
+import org.ksmt.utils.BvUtils.bvZero
+import org.ksmt.utils.BvUtils.minus
+import org.ksmt.utils.cast
+
+
+fun KContext.bias(format: KFpSort): KBitVecValue<KBvSort> {
+    val w = exponentWidth(format)
+    val binaryValue = "0".repeat(w - format.exponentBits.toInt() + 1) + "1".repeat(format.exponentBits.toInt() - 1)
+    val res = mkBv(binaryValue, w.toUInt())
+    check(res.sort.sizeBits.toInt() == w)
+    return res
+}
+
+
+fun KContext.minNormalExponent(format: KFpSort): KBitVecValue<KBvSort> {
+    // -(bias - 1)
+    val w = exponentWidth(format).toUInt()
+    return (bvZero(w) - (bias(format) - bvOne(w))).cast()
+}
+
+fun KContext.maxNormalExponent(format: KFpSort): KBitVecValue<KBvSort> {
+    return bias(format)
+}
+
+fun KContext.maxSubnormalExponent(format: KFpSort): KBitVecValue<KBvSort> {
+    return (bvZero(exponentWidth(format).toUInt()) - bias(format)).cast()
+}
+
+fun KContext.minSubnormalExponent(format: KFpSort): KBitVecValue<KBvSort> {
+    return (maxSubnormalExponent(format) - bvValue(
+      exponentWidth(format).toUInt(),
+      format.significandBits.toInt() - 2
+    )).cast()
+}
+
+fun KContext.boolToBv(expr: KExpr<KBoolSort>) = mkIte(expr, bvOne(), bvZero())
+fun KContext.bvToBool(expr: KExpr<KBvSort>) = mkIte(expr eq bvOne(), trueExpr, falseExpr)
+
+fun KContext.bvOne() = mkBv(1, 1u)
+fun KContext.bvZero() = mkBv(0, 1u)
+
+
+fun <T : KBvSort, S : KBvSort, F : KBvSort> KContext.mkBvConcatExpr(
+  arg0: KExpr<T>, arg1: KExpr<S>, arg2: KExpr<F>
+) = mkBvConcatExpr(mkBvConcatExpr(arg0, arg1), arg2)
+
+fun KContext.leadingOne(width: Int): KBitVecValue<KBvSort> {
+    val binaryValue = "1" + "0".repeat(width - 1)
+    return mkBv(binaryValue, width.toUInt())
+}
+
+fun <Fp : KFpSort> KContext.defaultSignificand(sort: Fp) = leadingOne(sort.significandBits.toInt())
+fun <Fp : KFpSort> KContext.defaultExponent(sort: Fp): KBitVecValue<KBvSort> {
+    return bvZero(exponentWidth(sort).toUInt()).cast()
+}
+
+fun significandWidth(format: KFpSort): Int = format.significandBits.toInt()
+
+// Get the number of bits in the unpacked format corresponding to a
+// given packed format.  These are the unpacked counter-parts of
+//  format.exponentWidth() and format.significandWidth()
+
+fun exponentWidth(format: KFpSort): Int {
```

Comment:
Please, use doc-style multiline comments `/**...*/`

Reply:
Let's rename all the related functions in a some more clear way: For example, `unpackedExponentWidth`